test/scripts/models/contents/page.js

```diff
@@ -0,0 +1,32 @@
+describe('test page model', function () {
+  'use strict';
+  var Page,
+    expect = chai.expect;
+
+  beforeEach(function (done) {
+    require(['cs!models/contents/page'], function () {
+      Page = arguments[0];
```

Comment:
Rather than setting this to `arguments[0]`, `Page` could just be moved up into the function params.